src/workflow/runtime/WorkflowRuntime.ts

```diff
@@ -0,0 +1,112 @@
+/*
+Copyright 2024 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import * as grpc from "@grpc/grpc-js";
+import { ActivityContext, OrchestrationContext, TaskHubGrpcWorker } from "@microsoft/durabletask-js";
+import { TWorkflow } from "../../types/workflow/Workflow.type";
+import { TWorkflowActivity } from "../../types/workflow/Activity.type";
+import { TInput, TOutput } from "../../types/workflow/InputOutput.type";
+import WorkflowActivityContext from "./WorkflowActivityContext";
+import WorkflowContext from "./WorkflowContext";
+import { generateInterceptors } from "../internal/ApiTokenClientInterceptor";
+import { getFunctionName } from "../internal";
+
+export default class WorkflowRuntime {
+  private worker: TaskHubGrpcWorker;
+  constructor(hostAddress?: string, options?: grpc.ChannelOptions) {
```

Comment:
This hostAddress is the Dapr sidecar, right? Can you please check how DaprClient infers the host based on different env vars if it's not provided? We can do the same for consistency. Same for WorkflowClient.

Reply:
update to use `Settings.getDefaultHost()` to get the default host to be consistency across the dapr project.